dmoj/executors/LEAN4.py

```diff
@@ -10,6 +10,8 @@ def main : IO Unit := do
   let line ← cin.getLine
   IO.println line
 """
+    nproc = -1
```

Comment:
Do we really need `nproc = -1`? Specifically, are we sure the number of spawned processes isn't bounded by some large-but-still-reasonable integer?